### PR TITLE
switch to new @from_network synthax in docs

### PR DESCRIPTION
**Pull request by OkonSamuel**
addresses #608

### CODE REVIEW

File: docs/src/transformers.md [hunk 1]
```diff
@@ -98,16 +98,29 @@ y1 = predict(ridgeM, W)
 knnM = machine(knn, W, y)
 y2 = predict(knnM, W)
 
-mach = machine(averager)
-yhat = transform(mach, y1, y2)
+mach= machine(averager)
+yhat = transform(averagerM, y1, y2)
 ```
```
Comment (ablaom) [inline docs/src/transformers.md:103]: @OkonSamuel Many thanks for this!

I guess you mean to replace `mach` on line 101 with `averagerM` as well.

I know it's a pain, but I always test all code that isn't automatically run  by `Documenter.jl` to catch such things.  The alternative is to change the "```julia" block to a "```example???" block, but this is also sometimes more hassle, and I had some issues with Documenter.jl early on  (so it's a bit of a mish-mash or evaluated and non-evaluated code blocks at the moment).

Comment (OkonSamuel) [reply docs/src/transformers.md:103]: > I guess you mean to replace `mach` on line 101 with `averagerM` as well.

Yeah. fixed that now.

**Comment by OkonSamuel** [comment]
Well this can be merged after a new release of **MLJBase** is created.

**Comment by ablaom** [comment]
I'm confused. This should work with MLJBase since 0.14.0, no?

**Comment by OkonSamuel** [comment]
> This should work with MLJBase since 0.14.0, no?

It doesn't. `yhat = transform(averagerM, y1, y2)` throws an error.

**Comment by ablaom** [comment]
@OkonSamuel Please base all PR's onto dev branch, even doc fixes. Thanks

**Comment by OkonSamuel** [comment]
> even doc fixes

noted. Sorry about that

**Pull request merged**